.gitignore

```diff
@@ -37,7 +37,9 @@ yarn-error.log*
 
 # gen
 /abis/types
-/types/generated
+/types/generated/abis/*
+/types/generated/graphql/*
+!/types/generated/graphql/twabs.ts
```

Comment:
since we do need to have the key in the .env is it worth not committing this file and setting a secret for render/vercel to use?